src/wallet/rpcwallet.cpp

```diff
@@ -4839,6 +4839,37 @@ static RPCHelpMan walletdisplayaddress()
 }
 #endif // ENABLE_EXTERNAL_SIGNER
 
+static RPCHelpMan migratewallet()
+{
+    return RPCHelpMan{"migratewallet",
+        "\nMigrate the wallet to a descriptor wallet.\n"
+        "A new wallet backup will need to be made.",
+        {},
+        RPCResult{RPCResult::Type::NONE, "", ""},
+        RPCExamples{
+            HelpExampleCli("migratewallet", "")
+            + HelpExampleRpc("migratewallet", "")
+        },
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    if (!wallet) return NullUniValue;
+    CWallet* const pwallet = wallet.get();
+
+    EnsureWalletIsUnlocked(*pwallet);
+
+    WalletContext& context = EnsureWalletContext(request.context);
+
+    bilingual_str error;
+    std::vector<bilingual_str> warnings;
```

Comment:
is this ignored?

Reply:
Removed, as well as in the `Migrate*` functions that aren't actually using `warnings`.